### PR TITLE
Add Missing Compendium Textures

### PR DESCRIPTION
**Pull request by jocelynbachmann**
Adds the missing plant compendium textures (except pinus contorta var latifolia which I couldn't find any better version for).

### CODE REVIEW

File: mods/magnify/magnify_flowers_nodes/init.lua [hunk 1]
```diff
@@ -139,48 +135,45 @@ local pearl = {
...
     sci_name = "Boletus edulis",
     com_name = "King bolete",
     fam_name = "Boletaceae",
-    cons_status = "Unranked",
-    status_col = "#808080", -- Unranked
+    cons_status = {ns_bc = "NR"},
     height = "8 to 25 centimeters tall",
     bloom = "The caps might have a white bloom on them - a dusty white powdered substance that easily brushes off",         
     region = "the Pacific Northwest, often in hemlock (Tsuga heterophylla), spruce (Picea sitchensis), pine (Pinus spp.) and fir (Abies spp.) forests",        
     texture = "magnify_flowers_boletus_edulis.jpeg",
```
Comment (pauldpickell) [inline mods/magnify/magnify_flowers_nodes/init.lua]: File extension is `jpg` not `jpeg`. Image does not load in game.

File: mods/magnify/magnify_flowers_nodes/init.lua [hunk 2]
```diff
@@ -191,22 +184,20 @@ local waterlily = {
...
     sci_name = "Amanita muscaria",
     com_name = "Fly agaric",
     fam_name = "Amanitaceae",
-    cons_status = "Unlisted",
-    --status_col = "", -- default
+    cons_status = {ns_bc = "Unlisted"},
     height = "7 to 20 centimeters tall",
     bloom = "Produces smooth, white, ellipsoid, inamyloid spores",
     region = "various areas in the Northern Hemisphere, including California",
     texture = "magnify_flowers_amanita_muscaria.jpeg",
```
Comment (pauldpickell) [inline mods/magnify/magnify_flowers_nodes/init.lua]: File extension is `jpg` not `jpeg`. Image does not load in game.

**Review by pauldpickell — CHANGES_REQUESTED** [review]
Looks good. Some minor issues with file extensions need to be fixed before merging. Noting here that the `default:jungle_tree` is mismatched with __Alnus rubra__. I am not sure which species the jungle tree is supposed to be, but we either need to make a new texture set and model for __Alnus rubra__ or find a new species and populate the correct information. 

@ginny-hang any ideas for which species this jungle tree texture might best represent?

**Comment by ginny-hang** [comment]
> Looks good. Some minor issues with file extensions need to be fixed before merging. Noting here that the `default:jungle_tree` is mismatched with **Alnus rubra**. I am not sure which species the jungle tree is supposed to be, but we either need to make a new texture set and model for **Alnus rubra** or find a new species and populate the correct information.
> 
> @ginny-hang any ideas for which species this jungle tree texture might best represent?

How about we change it to **Bigleaf Maple** which is on the to-do list? [When it grows in the forest, it develops a narrow crown that is supported by a stem free of branches for half its length. Those growing in the open have a broad crown which is supported by a few large, spreading limbs.](https://www.for.gov.bc.ca/hfd/library/documents/treebook/bigleafmaple.htm)  This description is more matched to `default:jungle_tree` based on how it looks in Minetest. We would then need a new texture for **Alnus Rubra**.
Screenshot_20220804-083402

**Pull request merged**